COMPILING

```diff
@@ -134,46 +134,38 @@ Follow these instructions:
 COMPILATION ON WINDOWS
 ----------------------
 
-There are two options: compilation using g++ from Cygwin, or using
-Visual Studio's compiler. As Cygwin has significant overhead during
-process creation, we advise you use Visual Studio.
+There are three compilation options: in Cygwin with the mingw cross compiler,
+or in Cygwin with g++, or with Visual Studio's compiler. As Cygwin has
+significant overhead during process creation, we recommend Visual Studio.
 
 Follow these instructions:
 
 0) You need a C/C++ compiler, Flex and Bison, GNU tar, gzip2,
-   GNU make, and patch. The GNU Make needs to be version 3.81 or
+   GNU make, wget, and patch. The GNU Make needs to be version 3.81 or
    higher.  If you don't already have the above, we recommend you install
    Cygwin.
 
-1) You need a SAT solver (in source). We recommend MiniSat2. Using a
-   browser, download from
+1) Get the CBMC source via
 
-   http://ftp.debian.org/debian/pool/main/m/minisat2/minisat2_2.2.1.orig.tar.gz
-
-   and then unpack with
+   git clone https://github.com/diffblue/cbmc cbmc-git
 
-   tar xfz minisat-2.2.1.tar.gz
-   mv minisat minisat-2.2.1
-   cd minisat-2.2.1
-   patch -p1 < ../scripts/minisat-2.2.1-patch
+2A) To compile in Cygwin, install the mingw compiler or gcc-g++. You may also
+   need to adjust the relevant section in common to define CC and CXX.
 
-   The patch removes the dependency on zlib and prevents a problem
-   with a header file that is often unavailable on Windows.
+2B) To compile with Visual Studio, make sure you have at least Visual Studio
+   version 12 (2013), and change the second line of config.inc to
 
-2A) To compile with Cygwin, install the mingw compilers, and adjust
-   the second line of config.inc to say
+   BUILD_ENV = MSVC
 
-   BUILD_ENV = MinGW
+   Open the Visual Studio Command prompt, and then start the Cygwin shell with
```

Comment:
BUILD_ENV should be cygwin, which will be detected correctly if BUILD_ENV = AUTO and make is called from the cygwin shell.